arm-relay/2016-07-01/relay.json

```diff
@@ -0,0 +1,2144 @@
+    },
+    "NamespaceResource": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Realy/Namespaces"
+          ],
+          "x-ms-enum": {
+            "name": "NameSpaceResourceType",
+            "modelAsString": false
+          },
+          "description": "Resource type"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NamespaceProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of a Namespace resource."
+    },
+    "NamespaceProperties": {
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "The path of the relay."
+        },
+        "relayType": {
+          "type": "string",
+          "description": "Relay NameSpace Type.",
+          "enum": [
+            "NetTcp",
+            "Http"
+          ],
+          "x-ms-enum": {
+            "name": "Relaytype",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time the namespace was created."
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The time the namespace was updated."
+        },
+        "listenerCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of listeners for this relay. min : 1 and max:25 supported"
+        },
+        "requiresClientAuthorization": {
+          "type": "boolean",
+          "description": "true if client authorization is needed for this relay; otherwise, false."
+        },
+        "authorizationRules": {
+          "$ref": "#/definitions/AuthorizationRules"
+        },
+        "requiresTransportSecurity": {
+          "type": "boolean",
+          "description": "true if transport security is needed for this relay; otherwise, false."
+        },
+        "isDynamic": {
+          "type": "boolean",
+          "description": "true if the relay is dynamic; otherwise, false."
+        },
+        "userMetadata": {
+          "type": "string",
+          "description": "usermetadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."
+        },
+        "collectionName": {
+          "type": "string",
+          "description": "The name of the collection associated with the relay."
+        }
+      },
+      "description": "Properties of the Namespace."
```

Comment:
Since this is an input parameter object, please cross-check what the required parameters will be here